hq/app/aws/ec2/EC2.scala

```diff
@@ -120,4 +122,26 @@ object EC2 {
         UnknownUsage(ni.getDescription, ni.getNetworkInterfaceId)
       )
   }
+
+  private[ec2] def addVpcName (flaggedSgs : List[SGOpenPortsDetail], vpcs: Map[String, Vpc]): List[SGOpenPortsDetail] = {
```

Comment:
Try and format stuff correctly!

```scala
private[ec2] def addVpcName(flaggedSgs: List[SGOpenPortsDetail], vpcs: Map[String, Vpc]):
```

Elsewhere as well, I'll not call it out in every case.